### PR TITLE
Task/Contact Us Section

### PR DESCRIPTION
**Pull request by Teraaay**
## Related Links

- Task Link: https://www.notion.so/Project-UI-UX-Capstone-716c808e203c4cce85691202f6bd1eca?p=2623ce94f0b64f52bc2b28c1e704a0f8
- Route Link: N/A

## Description
- Make a floating action button
- Modal for chatbot

## Notes

N/A

## Screenshots
Large Screen
> image

Medium Screen
> image

Mobile Screen
> image

### CODE REVIEW

**Review by ume45876139 — APPROVED** [review]
# Working
## PC
<details>
<summary>Screenshot</summary>

スクリーンショット 2022-04-26 15 40 12

スクリーンショット 2022-04-26 15 40 37

</details>

## iPad

<details>
<summary>Screenshot</summary>

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-26 at 15 45 52

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-26 at 15 46 22

Simulator Screen Shot - iPad Pro (11-inch) (3rd generation) - 2022-04-26 at 15 46 34

</details>

## Mobile

<details>
<summary>Screenshot</summary>

IMG_7979

IMG_7980

IMG_7981

</details>

- Icon size is correct 
- Color is correct
- User can open chat
- User can send chat
- Icon position is correct
- User can close chatbox


# Testing data
test device: Chrome, Safari (Mac)
test time: 0.5h
comment: 0
iteration: 0
total test time: 0.5h

File: src/pages/components/contactUs.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        onClick={() => setIsOpen(true)}
+        src="/images/ChatFloatingButton.svg"
+        alt="chat floating button"
+      />
+      <img
+        className={isOpen ? 'floating-button' : 'hidden'}
+        onClick={() => setIsOpen(false)}
+        src="/images/CloseFloatingButton.svg"
+        alt="chat floating button"
+      />
```
Comment (egiegarciano14) [inline src/pages/components/contactUs.js]:
```suggestion
      <img
        className='floating-button'
        onClick={() => setIsOpen(!isOpen)}
        src={isOpen ? '/images/CloseFloatingButton.svg' : '/images/ChatFloatingButton.svg'}
        alt="chat floating button"
      />
```

Comment (egiegarciano14) [reply src/pages/components/contactUs.js]: I think we can do this :thinking:

**Comment by egiegarciano14** [comment]
Also make the files under pages folder to PascalCase :bow:

File: src/pages/components/contactUs.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { Fragment, useState } from 'react';
+import ChatBox from './chatBox';
+
+const ContactUs = () => {
+  const [isOpen, setIsOpen] = useState(false);
```
Comment (egiegarciano14) [inline src/pages/components/contactUs.js]: I think we can rename this to `isModalOpen` or `isChatBoxOpen` what do you think?

**Review by egiegarciano14 — APPROVED** [review]

File: src/assets/tailwind.output.css [hunk 3]
```diff
@@ -20,3 +20,10 @@
 }
 
 /* Custom Components */
+.floating-button {
```
Comment (roseaugusto) [inline src/assets/tailwind.output.css]: @Teraaay Please use separate css file for this and then import it here

File: src/pages/Home.js [hunk 4]
```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import MainHeadline from './components/MainHeadline';
+import ContactUs from './components/ContactUs';
 
 const Home = () => {
   return (
     <div>
       <MainHeadline />
+      Welcome!
```
Comment (roseaugusto) [inline src/pages/Home.js]: This text is okay to remove now

File: tailwind.config.js [hunk 5]
```diff
@@ -4,7 +4,10 @@ module.exports = {
     extend: {
       fontFamily: {
         allison: ['Allison Regular'],
-        nunito: ['Nunito'],
+        nunito: ['Nunito']
```
Comment (roseaugusto) [inline tailwind.config.js]: Please retain the comma/s in the end of the of each property (please do the same with the rest)

File: src/pages/components/ChatBox.js [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+          : 'hidden'
+      }
+    >
+      <div className="h-16 w-100 flex justify-between items-center bg-gradient-to-b from-sky-900 via-sky-700 to-mocha-500 rounded-t-lg px-3">
+        <img
+          className="rounded-full w-10"
+          src="/images/RoundOrangeLogo.svg"
+          alt="logo"
+        />
+        <svg
```
Comment (roseaugusto) [inline src/pages/components/ChatBox.js]: Please put this on a separate svg files. Make icons folder inside components `/components/icons/CloseIcon.js` then just pass the functions as a prop there.

Also one one smol favor. Please put hover effect on close icon. Thanks 🙇‍♀️ 
image

**Review by charlotteds — APPROVED** [review]

**Comment by roseaugusto** [comment]
@Teraaay maybe this was overlook haha. But please put hover style on close button, it's okay no tooltip. 🙇🏼 

https://user-images.githubusercontent.com/70557829/165447943-0ea3380c-3df2-4556-89b0-cd8c7807b715.mov

**Review by roseaugusto — APPROVED** [review]
Approved!

**Comment by CaneteJoash** [comment]
### Approved 
- able to Make a floating action button
- able to create Modal for chatbot
- Able to add hover style on close button.

### Screenshot
image

**Pull request merged**